components/layout/traversal.rs

```diff
@@ -143,7 +140,7 @@ fn construct_flows_at<'a, N>(context: &LayoutContext<'a>,
         el.mutate_data().unwrap().persist();
         unsafe { el.unset_dirty_descendants(); }
 
-        remove_from_bloom_filter(&context.shared.style_context, root, el);
+        thread_local.style_context.bloom_filter.maybe_pop(el);
```

Comment:
Just drop this line, I'm not really sure it's worth to keep it. The only reason it (IIRC) was there was to help a bit the parallel traversal, and (maybe) eat the bloom filter when the traversal ended (but that was only done in one of the workers, so oh well), or the generation didn't match (but now there's no generation, which is awesome).